lib/sensu-handler.rb

```diff
@@ -9,10 +9,10 @@ module Sensu
   class Handler
     include Sensu::Plugin::Utils
     include Mixlib::CLI
-    option :map_v2_event_into_v1,
-           description: 'Enable 2.x to 1.4 event mapping. Alternatively set envvar SENSU_MAP_V2_EVENT_INTO_V1=1.',
+    option :map_go_event_into_v1,
```

Comment:
should we rename this to `map_go_event_into_ruby`? now that our hands are not tied we may very well have a 2.x ruby release.

Reply:
renamed in latest commit